adapters/ix/ix.go

```diff
@@ -7,7 +7,6 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/golang/glog"
```

Comment:
should add adding [semgrep](https://github.com/prebid/prebid-server/tree/master/.semgrep/adapter) rule to catch such glog import in adapter package. 

Rule will catch instances where PR make changes to add log line in adapter code